$NetBSD: patch-configure,v 1.1 2013/07/10 09:11:05 wiz Exp $

Patch from Thomas Dickey <dickey@his.com>
addressing build issues on NetBSD with native X and SunOS.

--- configure.orig	2013-07-07 01:16:33.000000000 +0000
+++ configure
@@ -9225,7 +9225,7 @@ test -n "$verbose" && echo "	...before $
 
 echo "${as_me:-configure}:9226: testing ...before $LIBS ..." 1>&5
 
-LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s,-lXt ,-lXt -lX11 ," -e 's/  / /g'`
+LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s%-lXt %-lXt -lX11 %" -e 's%  % %g'`
 test -n "$verbose" && echo "	...after  $LIBS" 1>&6
 
 echo "${as_me:-configure}:9231: testing ...after  $LIBS ..." 1>&5
@@ -9497,7 +9497,7 @@ test -n "$verbose" && echo "	...before $
 
 echo "${as_me:-configure}:9498: testing ...before $LIBS ..." 1>&5
 
-LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s,-lXt ,-lXt $X_PRE_LIBS ," -e 's/  / /g'`
+LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s%-lXt %-lXt $X_PRE_LIBS %" -e 's%  % %g'`
 test -n "$verbose" && echo "	...after  $LIBS" 1>&6
 
 echo "${as_me:-configure}:9503: testing ...after  $LIBS ..." 1>&5
@@ -10267,7 +10267,7 @@ EOF
 	do
 		case "$LIBS" in
 		*-l$cf_trim_lib\ *-l$cf_trim_lib*)
-			LIBS=`echo "$LIBS " | sed -e 's/  / /g' -e 's/-l'$cf_trim_lib' //' -e 's/ $//'`
+			LIBS=`echo "$LIBS " | sed -e 's/  / /g' -e 's%-l'"$cf_trim_lib"' %%' -e 's/ $//'`
 			test -n "$verbose" && echo "	..trimmed $LIBS" 1>&6
 
 echo "${as_me:-configure}:10273: testing ..trimmed $LIBS ..." 1>&5
@@ -10436,7 +10436,7 @@ test -n "$verbose" && echo "	...before $
 
 echo "${as_me:-configure}:10437: testing ...before $LIBS ..." 1>&5
 
-LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s,$cf_first_lib ,$cf_first_lib $cf_pkgconfig_libs ," -e 's/  / /g'`
+LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s%$cf_first_lib %$cf_first_lib $cf_pkgconfig_libs %" -e 's%  % %g'`
 test -n "$verbose" && echo "	...after  $LIBS" 1>&6
 
 echo "${as_me:-configure}:10442: testing ...after  $LIBS ..." 1>&5
@@ -10449,7 +10449,7 @@ test -n "$verbose" && echo "	...before $
 
 echo "${as_me:-configure}:10450: testing ...before $LIBS ..." 1>&5
 
-LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s,$cf_first_lib ,$cf_first_lib -lXmu ," -e 's/  / /g'`
+LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s%$cf_first_lib %$cf_first_lib -lXmu %" -e 's%  % %g'`
 test -n "$verbose" && echo "	...after  $LIBS" 1>&6
 
 echo "${as_me:-configure}:10455: testing ...after  $LIBS ..." 1>&5
@@ -10460,7 +10460,7 @@ fi
 	do
 		case "$LIBS" in
 		*-l$cf_trim_lib\ *-l$cf_trim_lib*)
-			LIBS=`echo "$LIBS " | sed -e 's/  / /g' -e 's/-l'$cf_trim_lib' //' -e 's/ $//'`
+			LIBS=`echo "$LIBS " | sed -e 's/  / /g' -e 's%-l'"$cf_trim_lib"' %%' -e 's/ $//'`
 			test -n "$verbose" && echo "	..trimmed $LIBS" 1>&6
 
 echo "${as_me:-configure}:10466: testing ..trimmed $LIBS ..." 1>&5
@@ -10899,7 +10899,7 @@ test -n "$verbose" && echo "	...before $
 
 echo "${as_me:-configure}:10900: testing ...before $LIBS ..." 1>&5
 
-LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s,-lXt ,-lXt -lX11 ," -e 's/  / /g'`
+LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s%-lXt %-lXt -lX11 %" -e 's%  % %g'`
 test -n "$verbose" && echo "	...after  $LIBS" 1>&6
 
 echo "${as_me:-configure}:10905: testing ...after  $LIBS ..." 1>&5
@@ -11171,7 +11171,7 @@ test -n "$verbose" && echo "	...before $
 
 echo "${as_me:-configure}:11172: testing ...before $LIBS ..." 1>&5
 
-LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s,-lXt ,-lXt $X_PRE_LIBS ," -e 's/  / /g'`
+LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s%-lXt %-lXt $X_PRE_LIBS %" -e 's%  % %g'`
 test -n "$verbose" && echo "	...after  $LIBS" 1>&6
 
 echo "${as_me:-configure}:11177: testing ...after  $LIBS ..." 1>&5
@@ -15315,7 +15315,7 @@ test -n "$verbose" && echo "	...before $
 
 echo "${as_me:-configure}:15316: testing ...before $LIBS ..." 1>&5
 
-LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s,-lXft ,-lXft $cf_pkgconfig_libs ," -e 's/  / /g'`
+LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s%-lXft %-lXft $cf_pkgconfig_libs %" -e 's%  % %g'`
 test -n "$verbose" && echo "	...after  $LIBS" 1>&6
 
 echo "${as_me:-configure}:15321: testing ...after  $LIBS ..." 1>&5
@@ -15328,7 +15328,7 @@ test -n "$verbose" && echo "	...before $
 
 echo "${as_me:-configure}:15329: testing ...before $LIBS ..." 1>&5
 
-LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s,-lXft ,-lXft -lfontconfig ," -e 's/  / /g'`
+LIBS=`echo "$LIBS" | sed -e "s/[ 	][ 	]*/ /g" -e "s%-lXft %-lXft -lfontconfig %" -e 's%  % %g'`
 test -n "$verbose" && echo "	...after  $LIBS" 1>&6
 
 echo "${as_me:-configure}:15334: testing ...after  $LIBS ..." 1>&5
